widget/tabcontainer.go

```diff
@@ -163,6 +163,25 @@ func (t *TabContainer) buildTabBar(buttons []fyne.CanvasObject) *fyne.Container
 	return tabBar
 }
 
+
+// Append adds a new CanvasObject to the end of the group
+func (t *TabContainer) Append(item *TabItem) {
+	r := Renderer(t).(*tabContainerRenderer)
```

Comment:
Renderer() is deprecated.
If you need to get access to a renderer use cache.Renderer(t).

Reply:
p.s. can you also delete the old commented out code which this replaces? :)

Reply:
Done:)